kokoro/continuous_mac.sh

```diff
@@ -8,6 +8,12 @@ set -x
 gcloud components update
 gcloud components install app-engine-java
 
+# Use adopt openjdk 8u202 until kokoro updates it's macos images
```

Comment:
Add the bug number? The comment will become questionable if looked at a couple years later.

Reply:
Makes sense. Will do.